### PR TITLE
Publicar "Normalização de Dados Textuais com Python"

### PR DESCRIPTION
**Pull request by DanielAlvesLABDH**
Publicar "Normalização de Dados Textuais com Python"

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by anisa-hawes** [comment]
I've adjusted one of the external links in this lesson. It directed to the dynamic Unicode homepage <https://home.unicode.org/>, which couldn't be archived with perma.cc.

In the context of the sentence, I thought that this link to the Quick Start Guide would be just as suitable [https://home.unicode.org/technical-quick-start-guide/](https://perma.cc/7ACH-KCDN).

If I understand correctly, the sentence reads: _If you work with fonts in more than one language, you need to learn a little more about the [Unicode] standard_

File: pt/licoes/normalizacao-dados-textuais-python.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Está claro que a nossa lista precisará de uma limpeza antes de conseguirmos utilizá-la para contar frequências. Em linha com as práticas estabelecidas em [De HTML para Lista de Palavras (parte 1)](/pt/licoes/HTML-lista-palavras-1), vamos tentar descrever o nosso algoritmo em português primeiro. Desejamos saber a frequência de cada palavra com sentido que aparece na transcrição do julgamento. Desse modo, as etapas envolvidas podem ser semelhantes a estas:
+
+- Converter todas as palavras para letras minúsculas de modo que "BENJAMIN" e "benjamin" sejam contabilizadas como a mesma palavra
+- Remover quaisquer caracteres estranhos ou incomuns
+- Contar o número de vezes que cada palavra aparece
+- Remover palavras excessivamente comuns como "it", "the", "and", etc.
+
+## Converter para Minúsculas
+
+Tipicamente tokens são convertidos em letras minúsculas ao contar frequências, então faremos isso através do método de string `lower` que foi introduzido em [Manipular strings com Python][]. Já que este é um método de string, devemos aplicá-lo à string `text` no programa `html-to-list1.py`. Ajuste `html-to-list1.py` adicionando a *string tag* `lower()` ao final da string `text`.
```
Comment (ericbrasiln) [inline pt/licoes/normalizacao-dados-textuais-python.md]: Aqui na linha 69 não consta o link no trecho 
>que foi introduzido em [Manipular strings com Python][].

Além de incluir o link, é preciso substituir `[]` por `()`

**Review by ericbrasiln — CHANGES_REQUESTED** [review]
O código está funcionando perfeitamente. Mas há uma correção necessária na linha 69.

File: pt/licoes/normalizacao-dados-textuais-python.md (hunk 1 above)
Comment (anisa-hawes) [reply pt/licoes/normalizacao-dados-textuais-python.md]: Apologies, @ericbrasiln! I missed this! I can add it in now.

**Review by ericbrasiln — APPROVED** [review]
Great! 🚀

**Comment by DanielAlvesLABDH** [comment]
Obrigado a ambos! Já estamos próximos das 30! 🤩🥳

**Comment by anisa-hawes** [comment]
Maravilhoso @DanielAlvesLABDH! Let us know when you are at 29, I imagine @GeeAlmeida will be keen to prepare some celebrations on Twitter!

**Comment by DanielAlvesLABDH** [comment]
This is the 29th @anisa-hawes

**Pull request merged**